samples/DotNet/Microsoft.Azure.ServiceBus/BasicSendReceiveUsingQueueClient/Program.cs

```diff
@@ -0,0 +1,105 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace BasicSendReceiveUsingQueueClient
+{
+    using Microsoft.Azure.ServiceBus;
+    using Microsoft.Azure.ServiceBus.Primitives;
+    using System;
+    using System.Text;
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    class Program
+    {
+        const string ServiceBusConnectionString = "{ServiceBus connection string}";
```

Comment:
@vinaysurya leave it as is. Just because I've done it one way, don't mean the suggestion is going to be good for this sample. If there's friction, people will let know. I don't have a strong opinion on this 🙂